### DISCUSSION TITLE
MSSQL - Escape certain characters in connection string

### DISCUSSION THREAD

**Discussion by acikgozb** [kndndrj/nvim-dbee#96] [Q&A]
Hey there, first of all thanks for the plugin.

I really want to switch from Jetbrains' DataGrip to this plugin, and currently the major blocker for me is the special characters in the password.

I have a password that contains `@`. When I try to connect the DB (mssql), I get this error:

`c.GetStructure: c.db.Conn: lookup rA: no such host`

I believe the reason why it prints out `rA` is that these are the next 2 characters in the password after `@`. 
However I'm not so sure about this.

So, is there a way to escape this character inside the password since it's also used before the host as well?

**Comment by RayJameson** [comment]
@acikgozb you can use python for that
```python
from urllib.parse import quote

print(quote("your password here"))
```
